contracts/Governance/AccessControlledV5.sol

```diff
@@ -26,31 +24,6 @@ contract AccessControlledV5 is InitializableV5, Ownable2StepUpgradeableV5 {
     /// @notice Emitted when access control manager contract address is changed
     event NewAccessControlManager(address oldAccessControlManager, address newAccessControlManager);
 
-    function __AccessControlled_init(address accessControlManager_) internal onlyInitializing {
-        __Ownable2Step_init();
-        __AccessControlled_init_unchained(accessControlManager_);
-    }
-
-    function __AccessControlled_init_unchained(address accessControlManager_) internal onlyInitializing {
-        _setAccessControlManager(accessControlManager_);
-    }
-
-    /**
-     * @notice Sets the address of AccessControlManager
-     * @dev Admin function to set address of AccessControlManager
-     * @param accessControlManager_ The new address of the AccessControlManager
-     */
-    function setAccessControlManager(address accessControlManager_) external onlyOwner {
-        _setAccessControlManager(accessControlManager_);
-    }
-
-    /**
-     * @notice Returns the address of the access control manager contract
-     */
-    function accessControlManager() external view returns (IAccessControlManagerV5) {
-        return _accessControlManager;
-    }
```

Comment:
Sorry for the misleading Slack post, maybe we could keep this external view for the transparency purposes, it won't harm IMO